frontEnd/src/main/java/olumofe/raphael/frontEnd/CarController.java

```diff
@@ -0,0 +1,35 @@
+package olumofe.raphael.frontEnd;
+
+import org.springframework.web.bind.annotation.*;
+import starterProject.domain.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+@org.springframework.stereotype.Controller
+
+public class CarController {
+
+    Transmission automatic = Transmission.builder().automatic(true).build();
+    Wheels parelli = Wheels.builder().wheels(4).build();
+    Engine v12 = Engine.builder().cylinders(12).build();
+    Doors verticalDoors = Doors.builder().doorNumbers(2).build();
+    @PostMapping("/createCarList")
```

Comment:
I wouldn't structure a REST API like that. You want to structure it in a way that is extensible to other operations on an object such as the car.

In this case I would expect you POST/PUT with a list to an endpoint with the path `/cars`.

Note the difference between POST and PUT as well. POST _should_ override a car list, but PUT should update it with new values.
Read this for more info.
https://restfulapi.net/resource-naming/